packages/docs/src/examples/border-radius/misc-removing-border-radius.vue

```diff
@@ -2,21 +2,55 @@
 <template>
   <v-row justify="center">
     <v-col cols="12">
-      <div
-        class="pa-4 text-center secondary rounded-0"
-        v-text="`.rounded-0`"
-      ></div>
-    </v-col>
-    <v-col
-      cols="12"
-      md="3"
-      v-for="value in ['t', 'r', 'b', 'l', 'tl', 'tr', 'br', 'bl']"
-      :key="value"
-    >
-      <div
-        :class="`pa-4 text-center secondary rounded-lg rounded-${value}-0`"
-        v-text="`rounded-${value}-0`"
-      ></div>
+      <div class="pa-4 text-center secondary rounded-0">.rounded-0</div>
+    </v-col>
+
+    <v-col cols="12" md="3">
+      <div class="pa-4 text-center secondary rounded-lg rounded-t-0">
+        rounded-t-0
```

Comment:
```suggestion
        .rounded-t-0
```

Reply:
this should be done to all examples. either prepend a **.** demonstrating a class or don't.